tests/comparisons/Model/testBakeTablePersistent.php

```diff
@@ -0,0 +1,125 @@
+<?php
+declare(strict_types=1);
+
+namespace Bake\Test\App\Model\Table;
+
+use Cake\ORM\Query;
+use Cake\ORM\RulesChecker;
+use Cake\Validation\Validator;
+
+/**
+ * Items Model
+ *
+ * @property \Bake\Test\App\Model\Table\UsersTable&\Cake\ORM\Association\BelongsTo $Users
+ * @property \Bake\Test\App\Model\Table\TodoTasksTable&\Cake\ORM\Association\HasMany $TodoTasks
+ * @property \Bake\Test\App\Model\Table\TodoLabelsTable&\Cake\ORM\Association\BelongsToMany $TodoLabels
+ *
+ * @method \Bake\Test\App\Model\Entity\Item newEmptyEntity()
+ * @method \Bake\Test\App\Model\Entity\Item newEntity(array $data, array $options = [])
+ * @method \Bake\Test\App\Model\Entity\Item[] newEntities(array $data, array $options = [])
+ * @method \Bake\Test\App\Model\Entity\Item get($primaryKey, $options = [])
+ * @method \Bake\Test\App\Model\Entity\Item findOrCreate($search, ?callable $callback = null, $options = [])
+ * @method \Bake\Test\App\Model\Entity\Item patchEntity(\Cake\Datasource\EntityInterface $entity, array $data, array $options = [])
+ * @method \Bake\Test\App\Model\Entity\Item[] patchEntities(iterable $entities, array $data, array $options = [])
+ * @method \Bake\Test\App\Model\Entity\Item|false save(\Cake\Datasource\EntityInterface $entity, $options = [])
+ * @method \Bake\Test\App\Model\Entity\Item saveOrFail(\Cake\Datasource\EntityInterface $entity, $options = [])
+ * @method \Bake\Test\App\Model\Entity\Item[]|\Cake\Datasource\ResultSetInterface|false saveMany(iterable $entities, $options = [])
+ * @method \Bake\Test\App\Model\Entity\Item[]|\Cake\Datasource\ResultSetInterface saveManyOrFail(iterable $entities, $options = [])
+ * @method \Bake\Test\App\Model\Entity\Item[]|\Cake\Datasource\ResultSetInterface|false deleteMany(iterable $entities, $options = [])
+ * @method \Bake\Test\App\Model\Entity\Item[]|\Cake\Datasource\ResultSetInterface deleteManyOrFail(iterable $entities, $options = [])
+ *
+ * @mixin \Cake\ORM\Behavior\TimestampBehavior
+ */
+class ItemsTable extends ItemsPersistentTable
```

Comment:
I thought the classes that extended the persistent tables wouldn't have bake generated code? Right now it looks like you'd have all your associations defined twice, same with validation rules.

Reply:
classes that extends persistent tables are generated for every bake (if approved by user or used --force). Persistent class is baked only once and there are empty functions like validationDefault(), buildRules() etc. But this could be removed and would only empty persistent class. So baked code is generated only in main class, comments in persistent class with property definitions will be removed. As I said here https://github.com/cakephp/bake/issues/651 this approach worked ok for me because at the early stages of development when I add some functions to model and then add/rename some field, add relations I must always watch for those functions, class changes etc.. With that approach I can run bake without worrying that anything would be overwritten.

Reply:
Ok, I had it mixed up then. It would be good if the generated persistent classes included some doc block comments on how they differ from the bake managed classes and which methods should go where.